queries.sql

```diff
@@ -0,0 +1,61 @@
+SELECT * FROM animals WHERE animals.name like '%mon';
+SELECT name FROM animals WHERE animals.date_of_birth >= '2016-01-01' AND animals.date_of_birth <= '2019-12-31';
+SELECT name FROM animals WHERE animals.neutered = true AND animals.escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE animals.name = 'Agumon' OR animals.name = 'Pikachu';
+SELECT name, escape_attempts FROM animals WHERE animals.weight_kg > 10.5;
+SELECT * FROM animals WHERE animals.neutered = true;
+SELECT * FROM animals WHERE animals.name != 'Gabumon';
+SELECT * FROM animals WHERE animals.weight_kg >= 10.4 AND animals.weight_kg <= 17.3;
+
+-- queries for the second day's project
+
+
+BEGIN;
+
+UPDATE animals SET species = 'unspecified';
+
+SELECT * FROM animals;
+
+ROLLBACK;
+
+SELECT * FROM animals;
```

Comment:
- [ ] To verify that the changes persist after commit, we run `SELECT species from Animals`
![Screenshot from 2023-05-11 14-41-00](https://github.com/IsmailChangezi/vet-clinic/assets/46388113/1c397ada-6d6a-42ed-88d6-0625cdce2e0e)